tardis/model.py

```diff
@@ -268,6 +268,16 @@ def to_hdf(self, path_or_buf, path='', plasma_properties=None):
         None
 
         """
+        properties = ['t_inner', 'ws', 't_rads', 'v_inner', 'v_outer']
+        for p in properties:
+            output_value = getattr(self, p)
+            if hasattr(output_value, 'si'):
+                output_value = output_value.si.value
+            if np.isscalar(output_value):
+                output_value = [output_value]
+            data = pd.DataFrame(output_value)
```

Comment:
I think there is a high likelihood that this doesn't work. But all of the values in model have the same length.